docs/core/integration-quality-scale/index.md

```diff
@@ -74,6 +74,7 @@ The platinum tier has the following characteristics:
 - All source code follows all coding and Home Assistant integration standards and best practices and is fully typed with type annotations and clear code comments for better code clarity and maintenance.
 - A fully asynchronous integration code base ensures efficient operation.
 - Implements efficient data handling, reducing network and CPU usage.
+- Code owner approval is required on pull requests before merging.
```

Comment:
This is partly true, while we do respect it as much as possible, we will override it sometimes.

Reply:
Do you think it is worth rewording or clarifying? 
Assuming that the times we override are exceptions and not the norm, I would prefer to keep it simple.

Reply:
I disagree, the problem with writing it like you have done here, is that it sets expectations to our contributors that are not true. This will (and has led) to disagreements in the past.

For example, we had codeowners not agreeing we merged before they could approve, because it was written so strictly. I rather remove this.